incubator/sparkoperator/templates/spark-operator.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2017 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: sparkoperator
+  namespace: {{ .Values.operatorNamespace }}
+  labels:
+    app.kubernetes.io/name: sparkoperator
+    app.kubernetes.io/version: {{ .Values.operatorVersionLabel }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: sparkoperator
+      app.kubernetes.io/version: {{ .Values.operatorVersionLabel }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: sparkoperator
+        app.kubernetes.io/version: {{ .Values.operatorVersionLabel }}
+      initializers:
+        pending: []  
+    spec:
+      serviceAccountName: sparkoperator
+      containers:
+      - name: sparkoperator
+        image: {{ .Values.operatorImageName }}:{{ .Values.operatorImageTag }}
```

Comment:
I would force the operator image tag to be also version label ({{ .Values.operatorVersionLabel }}), what do you think @liyinan926 ?

Reply:
Yes, I agree.

Reply:
Thanks. Now fixed.